interpreter.py

```diff
@@ -16,8 +19,12 @@ def jitpolicy(driver):
     from rpython.jit.codewriter.policy import JitPolicy
     return JitPolicy()
 
+def get_location(interpreter, block, function):
```

Comment:
I'm nitpicking now, but why not call it `get_printable_location` which is standard RPython terminology.